ambari-web/app/templates/wizard/step3.hbs

```diff
@@ -19,141 +19,159 @@
-            </div>
-          {{/if}}
-          {{#unless disableHostCheck}}
-            <div {{bindAttr class=":alert :alert-warning view.status isWarningsBoxVisible::hidden"}} {{QAAttr "{view.status}"}}>
-              {{view.message}}
-              <a href="#" {{action hostWarningsPopup warnings target="controller"}} {{QAAttr "alert-link"}}>{{view.linkText}}</a>
-              {{#unless isBootstrapFailed}}
-                {{#unless isWarningsLoaded}}
-                  {{view App.SpinnerView data-qa="spinner"}}
-                {{/unless}}
-              {{/unless}}
-            </div>
-          {{/unless}}
+            </tfoot>
+          </table>
         </div>
       </div>
+      <div id="warningsSection">
+        {{#if hasMoreRegisteredHosts}}
+          <div {{bindAttr class=":alert :alert-warning"}} {{QAAttr "alert-warning"}}>
+            <a href="#" {{action registeredHostsPopup target="controller"}} {{QAAttr "alert-link"}}><span {{QAAttr "other-hosts-link"}}>{{view.registeredHostsMessage}}</span></a>
+          </div>
+        {{/if}}
+        {{#unless disableHostCheck}}
+          <div {{bindAttr class=":alert :alert-warning view.status isWarningsBoxVisible::hidden"}} {{QAAttr "{view.status}"}}>
+            {{view.message}}
+            <a href="#" {{action hostWarningsPopup warnings target="controller"}} {{QAAttr "alert-link"}}>{{view.linkText}}</a>
+            {{#unless isBootstrapFailed}}
+              {{#unless isWarningsLoaded}}
+                {{view App.SpinnerView data-qa="spinner"}}
+              {{/unless}}
+            {{/unless}}
+          </div>
+        {{/unless}}
+      </div>
     </div>
+  </div>
 </div>
-
 <div class="wizard-footer col-md-12">
   <div class="btn-area">
-    <button type="button" class="btn btn-default pull-left installer-back-btn" {{bindAttr disabled="isBackButtonDisabled"}} {{action back}} {{QAAttr "wizard-back"}}>
+    <button type="button"
+            class="btn btn-default pull-left installer-back-btn" {{bindAttr disabled="isBackButtonDisabled"}} {{action back}} {{QAAttr "wizard-back"}}>
       &larr; {{t common.back}}
       {{#if App.router.backBtnClickInProgress}}
         {{view App.SpinnerView tagName="span" classNames="service-button-spinner"}}
       {{/if}}
     </button>
-    <button type="button" class="btn btn-success pull-right" {{bindAttr disabled="isNextButtonDisabled"}} {{action submit target="controller"}} {{QAAttr "wizard-next"}}>
+    <button type="button"
+            class="btn btn-success pull-right" {{bindAttr disabled="isNextButtonDisabled"}} {{action submit target="controller"}} {{QAAttr "wizard-next"}}>
       {{#if App.router.nextBtnClickInProgress}}
         {{view App.SpinnerView tagName="span" classNames="service-button-spinner"}}
       {{/if}}
-      {{t common.next}} &rarr;
+      {{t installer.step3.next.configure.download}} &rarr;
```

Comment:
I know that the mockups show specific labels on the next buttons, but I'm not sure it's a great idea. For one thing, it makes it harder to reorder steps if needed. Also, we don't do the same thing to the back buttons. Finally, we have a very prominent navigation menu on the left, so it's not like the user won't know where they're going. If we really want the buttons to be this way, it would be better to make them automatically adjust the labels based on the next/previous steps. I can bring this up with Subhrajit.

Reply:
Not sure what to do here. Will wait to hear back from Subhrajit then.

Reply:
Changed it back to next